### ISSUE TITLE
Error: Unexpected character at index 16286

### ISSUE THREAD

**Issue by Weissnix4711** [theacodes/kicanvas#52]
Fails to open [this project](https://github.com/solokeys/solo2-hw). No idea why, but it would be nice to see this working. Other projects seem to open just fine. Tried both browsers I currently have installed. Logs below.

Chromium:

```
Error: Unexpected character at index 16286: ·
Context: 
    (fp_text user · (at 0.275 -0.95 90) (layer F.SilkS)
    at tokenize (kicanvas.js:929:430)
    at Nn.next (<anonymous>)
    at listify_tokens (kicanvas.js:935:82)
    at Os.next (<anonymous>)
    at Function.from (<anonymous>)
    at listify_tokens (kicanvas.js:935:192)
    at Os.next (<anonymous>)
    at Function.from (<anonymous>)
    at listify_tokens (kicanvas.js:935:192)
    at Os.next (<anonymous>)
setup_project @ kicanvas.js:1783
```

LibreWolf:

```
Error: Unexpected character at index 16286: ·
Context: 
    (fp_text user · (at 0.275 -0.95 90) (layer F.SilkS)
    Nn https://kicanvas.org/kicanvas/kicanvas.js:929
    Os https://kicanvas.org/kicanvas/kicanvas.js:935
    Os https://kicanvas.org/kicanvas/kicanvas.js:935
    Os https://kicanvas.org/kicanvas/kicanvas.js:935
    Os https://kicanvas.org/kicanvas/kicanvas.js:935
    Us https://kicanvas.org/kicanvas/kicanvas.js:935
    g https://kicanvas.org/kicanvas/kicanvas.js:935
    Ve https://kicanvas.org/kicanvas/kicanvas.js:936
    #n https://kicanvas.org/kicanvas/kicanvas.js:971
kicanvas.js:1783:978
    setup_project https://kicanvas.org/kicanvas/kicanvas.js:1783
```

**Comment by theacodes** [comment]
Is this a KiCAD 5 project?

**Comment by Weissnix4711** [comment]
Yes, I believe it is.

**Comment by Weissnix4711** [comment]
Ah, I see #34 now. Is kicad 5 support planned/do you have an ETA? Purely out of interest, I'm not trying to pressure you or anything.

**Comment by theacodes** [comment]
No, KiCAD 5 support isn't in scope right now.

**Comment by Weissnix4711** [comment]
No worries.

How hard do you think it would be to add support? Ie. Should I waste my time trying to learn about KiCAD's file formats, and understand this codebase, to try and add support myself? Or do you think I should not bother?

**Comment by theacodes** [comment]
V5 boards are relatively easier, but schematics are a vastly different
format and would be much harder. I wouldn't try to do it at this point in
KiCanvas's development.

**Comment by theacodes** [comment]
Closed, duplicate of #34
